starport/cmd/network_chain_list.go

```diff
@@ -0,0 +1,82 @@
+package starportcmd
+
+import (
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	"github.com/tendermint/starport/starport/pkg/cosmosaccount"
+	"github.com/tendermint/starport/starport/pkg/entrywriter"
+	"github.com/tendermint/starport/starport/services/network"
+)
+
+var LaunchSummaryHeader = []string{"launch ID", "chain ID", "source", "campaign ID"}
+
+// LaunchSummary holds summarized information about a chain launch
+type LaunchSummary struct {
+	LaunchID   string
+	ChainID    string
+	Source     string
+	CampaignID string
+}
+
+// NewNetworkChainList returns a new command to list all published chains on Starport Network
+func NewNetworkChainList() *cobra.Command {
+	c := &cobra.Command{
+		Use:   "list",
+		Short: "List published chains",
+		Args:  cobra.NoArgs,
+		RunE:  networkChainListHandler,
+	}
+	c.Flags().String(flagFrom, cosmosaccount.DefaultAccount, "Account name to use for sending transactions to SPN")
+	c.Flags().AddFlagSet(flagSetKeyringBackend())
+	c.Flags().AddFlagSet(flagSetHome())
+
+	return c
+}
+
+func networkChainListHandler(cmd *cobra.Command, args []string) error {
+	nb, err := newNetworkBuilder(cmd)
+	if err != nil {
+		return err
+	}
+	defer nb.Cleanup()
+
+	nb.Spinner.Stop()
```

Comment:
```suggestion
```

Reply:
We need to stop it at some point otherwise the output is
```
◢ Initializing... Launch ID 	Chain ID 	Source 				Campaign ID
1 		earth-1 	https://github.com/lubtd/earth 	1
2 		earth-1 	https://github.com/lubtd/earth 	2
3 		toast-0 	test.com 			no campaign
```